app/helpers/users_helper.rb

```diff
@@ -0,0 +1,27 @@
+module UsersHelper
+  def positions_list
+    Position.pluck(:name, :id)
+  end
+
+  def departments_list
+    arr = Department.pluck(:name, :id)
+    # arr << ["", nil]
+    arr
```

Comment:
You can remove 8 and 9 lines or uncomment.

Reply:
Done

Reply:
Branch WM6